elk/training/reporter.py

```diff
@@ -296,9 +299,10 @@ def score(
         cal_preds = pred_probs.gt(cal_thresh).squeeze(1).to(torch.int)
         raw_preds = pred_probs.gt(0.5).squeeze(1).to(torch.int)
 
-        auroc = float(roc_auc_score(labels.cpu(), pred_probs.cpu()))
-        cal_acc = cal_preds.eq(labels.reshape(-1)).float().mean()
-        raw_acc = raw_preds.eq(labels.reshape(-1)).float().mean()
+        tiled_labels = labels.repeat_interleave(pred_probs.shape[1])
```

Comment:
We should probably have some sort of a comment explaining what this is doing? Right now it's very opaque